runelite-client/src/main/java/net/runelite/client/plugins/tileindicators/TileIndicatorsPlugin.java

```diff
@@ -62,4 +81,34 @@ protected void shutDown() throws Exception
 	{
 		overlayManager.remove(overlay);
 	}
+
+	@Subscribe
+	public void onGameTick(GameTick e)
+	{
+		final WorldPoint playerPos = client.getLocalPlayer().getWorldLocation();
+
+		if (lastPlayerPosition != null)
+		{
+			if (!playerPos.equals(lastPlayerPosition))
+			{
+				lastTickPlayerMoved = client.getTickCount();
+			}
+			else if (lastTickPlayerMoved + 1 == client.getTickCount())
+			{
+				lastTimePlayerStoppedMoving = System.currentTimeMillis();
+			}
+		}
+
+		lastPlayerPosition = playerPos;
+	}
+
+	@Subscribe
+	public void onGameStateChanged(GameStateChanged e)
+	{
+		if (e.getGameState() == GameState.LOGIN_SCREEN)
+		{
+			// When true tile fadeout is enabled, this prevents the true tile from showing up right after logging in.
```

Comment:
This should also apply to world hops, and possibly disconnects?

Reply:
World hops do not cause your character to move so I don't think the tile will show up when you world hop. On a reconnects, it is advantageous for the tile to pop up if your character has moved since disconnecting.